lisa/operating_system.py

```diff
@@ -93,12 +124,20 @@ def is_windows(self) -> bool:
     def is_posix(self) -> bool:
         return self._is_posix
 
+    @property
+    def os_version(self) -> OsVersion:
+        if not self._os_version:
+            self._os_version = self._get_os_version()
+
+        return self._os_version
+
     @classmethod
     def _get_detect_string(cls, node: Any) -> Iterable[str]:
         typed_node: Node = node
         cmd_result = typed_node.execute(cmd="lsb_release -d", no_error_log=True)
         yield get_matched_str(cmd_result.stdout, cls.__lsb_release_pattern)
 
+        # It covers distros like ClearLinux too
```

Comment:
Earlier there was a different way to get the info for ClearLinux. Chi found from his testing that ClearLinux images are covered but wasn't sure which of these are covering ClearLinux. 
So I added just for fyi

Reply:
I feel it doesn't need to say, unless it can list mainly distro it handles. I guess the ClearLinux is not the most important one, which handled by this line.